toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/internal/WorldTrackingCameraController.kt

```diff
@@ -109,17 +127,18 @@ internal class WorldTrackingCameraController(private val onLocationDataSourceFai
      *
      * @since 200.7.0
      */
-    private fun updateCamera(location: Location) =
+    private fun updateCamera(location: Location) {
```

Comment:
Could still keep it as an expression?
```suggestion
    private fun updateCamera(location: Location) =
```